cloud-storage/onedrive/impl/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.sakaiproject.onedrive</groupId>
+        <artifactId>sakai-onedrive</artifactId>
+        <version>20-SNAPSHOT</version>
+    </parent>
+
+    <name>Sakai OneDrive Integration - IMPL</name>
+    <groupId>org.sakaiproject.onedrive</groupId>
+    <artifactId>sakai-onedrive-impl</artifactId>
+
+    <packaging>sakai-component</packaging>
+    <properties>
+        <deploy.target>components</deploy.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.sakaiproject.onedrive</groupId>
+            <artifactId>sakai-onedrive-api</artifactId>
+        </dependency>
+
+        <!--Quartz Dependencies-->
+        <dependency>
+            <groupId>org.quartz-scheduler</groupId>
+            <artifactId>quartz</artifactId>
+            <scope>provided</scope>
```

Comment:
This is declared in master as provided no need to override here.

Reply:
Ok, done